hyperspy/hspy.py

```diff
@@ -97,11 +97,9 @@ def create_model(signal, *args, **kwargs):
     A Model class
 
     """
-
-    from hyperspy._signals.eels import EELSSpectrum
-    from hyperspy.models.eelsmodel import EELSModel
-    from hyperspy.model import Model
-    if isinstance(signal, EELSSpectrum):
-        return EELSModel(signal, *args, **kwargs)
-    else:
-        return Model(signal, *args, **kwargs)
+    import warnings
+    warnings.warn(
+        "This function is deprecated and will be removed in HyperSpy 0.9. "
+        "Please use the equivalent `Signal.create_model` method "
+        "instead.", DeprecationWarning)
```

Comment:
I'm not sure if I get your point. I think that the issue was caused by the fact that [here](https://github.com/hyperspy/hyperspy/commit/9a4c307a4b103803c4099c92c47882cbe4cc555e#diff-e27c75df2dece36803d7c8ccc9166673L11) the filter default was changed to "ignore" and never changed back to "default".